### DISCUSSION TITLE
Finding release notes

### DISCUSSION THREAD

**Discussion by elenzil** [cnr-isti-vclab/meshlab#1143] [Ideas]
Hello !

Excited to see so much velocity in MeshLab lately! Such a great tool.

When MeshLab itself informed me that a new version was available, I had a lot of trouble finding the release notes,
which are here in the 'discussion' section of the git repository.
In fact, I navigated to the linked github Releases page,
and saw that 2021.05 had a substantial "change log" section while 2021.10 had only the comment "better cmake log",
so I decided this release wasn't super interesting to me.

Suggestions -

- Perhaps the release notes could consistently go with each release on the [releases page](https://github.com/cnr-isti-vclab/meshlab/releases).
- Alternatively, perhaps each release could have a static link to the [Discussions page](https://github.com/cnr-isti-vclab/meshlab/discussions) with a note "Check for release notes on the _discussions_ page" or similar.
- Even lower overhead would be to add a "check the discussions" link to the [releases portion of the readme](https://github.com/cnr-isti-vclab/meshlab#releases).

okay,
cheers!

**Comment by alemuntoni** [comment]
Hi, 
thanks for posting!
That was just my bad that I forgot to update the release note of github (it was update in the meshlab website though)!
Just updated it :)
